### PR TITLE
Put file fix

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

**Comment by jdoliner** [comment]
@sjezewski whenever you have a chance.

File: src/server/pfs/cmds/cmds.go [hunk 1]
```diff
@@ -485,40 +484,6 @@ Files can be read from finished commits with get-file.`,
 	var commitFlag bool
 	var inputFile string
 	// putFilePath is a helper for putFile
```
Comment (sjezewski) [inline src/server/pfs/cmds/cmds.go]: Looks like we can remove this comment too

**Comment by sjezewski** [comment]
image

**Comment by sjezewski** [comment]
One minor thing - that can be a separate PR since we want this to land this am

**Pull request merged**